.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "eslint-config-airbnb",
+    "plugin:prettier/recommended"
```

Comment:
I tried to compare what eslint-plugin-prettier does: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration vs. what https://github.com/facebook/create-react-app/blob/next/packages/eslint-config-react-app/index.js does. Given we already try to go down the airbnb ruleset in core, I think using the same here makes 100% sense.